src/components/Dashboard/PendingList/filings/ContinuationIn.vue

```diff
@@ -61,25 +42,6 @@ export default class ContinuationIn extends Vue {
   get entityType (): string {
```

Comment:
Line 40, above, is obsolete. You will also be able to remove the imports for Getter and useBusinessStore.

Reply:
Done